src/navigators/app-navigator.tsx

```diff
@@ -53,38 +54,65 @@ export type NavigatorParamList = {
 const Stack = createNativeStackNavigator<NavigatorParamList>();
 
 const AppStack = () => {
-  return (
-    <Stack.Navigator
-      screenOptions={{
-        headerShown: false,
-        animation: "slide_from_right",
-      }}
-      initialRouteName="main"
-    >
-      {/* auth stack */}
-      <Stack.Screen name="signin" component={SignInScreen} />
-      <Stack.Screen name="signup" component={SignUpScreen} />
-      <Stack.Screen name="otpVerification" component={OTPVerificationScreen} />
-      <Stack.Screen name="forgetPassword" component={ForgetPasswordScreen} />
-      <Stack.Screen name="resetPassword" component={ResetPasswordScreen} />
-
-      {/*  */}
-      <Stack.Screen name="main" component={MainScreen} />
-      <Stack.Screen name="notifications" component={NotificationScreen} />
-      <Stack.Screen name="suggestions" component={SuggestionsScreen} />
-      <Stack.Screen name="publicProfile" component={PublicProfileScreen} />
+  const { user, loading: userLoading } = useAppSelector((state: RootState) => state.auth);
 
-      <Stack.Screen name="usermessaging" component={UserMessagingScreen} />
-      <Stack.Screen name="blockedusers" component={BlockedUsersScreen} />
-      <Stack.Screen name="settings" component={SettingsScreen} />
-      <Stack.Screen name="editprofile" component={EditProfileScreen} />
-      <Stack.Screen name="appsettings" component={AppSettingsScreen} />
-      <Stack.Screen name="contactUs" component={ContactUsScreen} />
-      <Stack.Screen name="changePassword" component={ChangePasswordScreen} />
+  // console.log("user === ", user);
```

Comment:
Remove unnecessary comments from here and clean up the if / else logic without use of return keyword.